js/index.js

```diff
@@ -1,7 +1,63 @@
-// Iteration 1: Names and Input
+console.log("I'm ready")
 
+// Iteration 1: Names and Input
+const hacker1 = "Marie";
+console.log(`"The driver's name is ${hacker1}"`);
+const hacker2 = "Yosra";
+console.log(`"The navigator's name is ${hacker2}"`);
 
 // Iteration 2: Conditionals
+if(hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+} else if(hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+} else {
+    console.log(`Wow, you both have equally long names, ${hacker1} characters!`)
+}
+
+// Iteration 3: Loops
+let newDriverName = "";
+for (let i = 0; i < hacker1.length; i++) {
+    newDriverName += hacker1[i].toUpperCase() + " "
+}
+console.log(newDriverName);
+
+let newNavigatorName = "";
+for (let i = hacker2.length-1; i >= 0; i--) {
+    newNavigatorName += hacker2[i]
+}
+console.log(newNavigatorName);
+
+if (hacker1.localeCompare(hacker2) === -1){
```

Comment:
A few things here with locale compare:
Some browsers can return -2 and some -1 (info can be found at https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/localeComparet) so it is more safe to write:
if (hacker1.localeCompare(hacker2) < 0) 

firstVariable.localeCompare(secondVariable) returns a negative number if the firstVariable is longer and